src/modules/person/api/person.controller.spec.ts

```diff
@@ -712,6 +714,7 @@ describe('PersonController', () => {
             const lockUserBodyParams: LockUserBodyParams = {
                 lock: false,
                 locked_from: '2024-01-01T00:00:00Z',
+                locked_until: new Date(),
             };
```

Comment:
Just noticing this now: What is `locked_from` supposed to be? I assumed this was a free-form locking-reason, but here you pass it an ISO-date